palette/src/oklab/random.rs

```diff
@@ -0,0 +1,91 @@
+use super::Oklab;
+use crate::num::Real;
+use rand::distributions::uniform::{SampleBorrow, SampleUniform, UniformSampler};
+use rand::distributions::{Distribution, Standard, Uniform};
+use rand::Rng;
+use std::ops::{Mul, Sub};
+
+impl<T> Distribution<Oklab<T>> for Standard
+where
+    T: Real + Mul<Output = T> + Sub<Output = T>,
+    Standard: Distribution<T>,
+{
+    // `a` and `b` both range from (-1.0, 1.0)
+    // TODO: the choice for a and b is random and rather bad:
+    //  1. a and b are unlimited. Oklab can express the whole electro-magnetic spectrum
+    //  2. Oklab is a perceptual color space. It would make sense to limit random
+    //  values to the limits of human perception.
+    //  https://bottosson.github.io/posts/oklab/#luo-rigg-dataset-and-full-gamut
+    //  shows, that at least for some L, a should not be greater than 0.5, to
+    //  avoid leaving the perceivable gamut. Though it could go to -2.5.
+    //  3. If people want random sRGB values: Expressing the sRGB bounds in Oklab is
+    //  beyond my abilities. However, it is not necessary either. It can be done in
+    //  Okhsv, Okhsl or Okhwb. Maybe we should  not offer the ability in Oklab and
+    //  encourage sampling in the color spaces that are limited to sRGB gamut.
```

Comment:
These are very good points. I don't have it fresh in mind at the moment but there is an option to sample a specific range. I think that still makes sense to keep, since I don't think there's any reason to prevent that. If a default alternative is still necessary (again, I would have to take a look later), this could be kept as is, since it allows easy scaling to other ranges by multiplication. Similar to a random function returning 0.0-1.0.